docs/changelog.md

```diff
@@ -1,6 +1,8 @@
 dev
 
 - [docs] Fix the command for [installing development version](https://pypa.github.io/pipx/installation/#install-pipx-development-versions). (#801)
+- Improve the behaviour of `shlex.split` on Windows (#794)
```

Comment:
This is technically correct, but users won't know what to make of it. Could you mention what this fixes from an end user perspective?